### ISSUE TITLE
Help the user plot the evolution of a specific element

### ISSUE THREAD

**Issue by ilyasst** [ets-lipec/muDIC#3]
Currently, the code does not allow the user to easily extract and plot the evolution of displacement, engineering strain or other for a specific node.
This issue aims at providing the user with an easy way to do so.

Here is how I propose you implement this:
1. Allow the user to specify a new `setting` for `dic.DICInput` (see https://github.com/ets-lipec/muDIC/blob/master/Examples/quick_start.py#L29): `settings.element_map = True`. Once activated this flag will show the nodes map to the user superposed on top of the image experiment
A combination of 
![image](https://user-images.githubusercontent.com/443955/89205186-52847300-d585-11ea-89a5-71e6f790f19e.png)
and this for example
![ut_151](https://user-images.githubusercontent.com/443955/89205258-71830500-d585-11ea-98cc-2d1a5a462c27.jpg)
2. Allow the user to use a function in `viz` that can be used in this way: `viz.save_node(node=42, field="displacement", component = (1,1))`. This function will save a plot that shows the evolution of displacement for component 1,1 for node number 42. (similar to what is shown in the last video of MacroDIC_forever where we can choose a point on the image and plot its evolution)

That's it !

**Comment by ilyasst** [comment]
https://github.com/ets-lipec/muDIC/pull/10 partly solved this. The code can present the nodes and their numbers but it does not show the specimen and location of each node at the same time
![Figure](https://user-images.githubusercontent.com/443955/90989656-71b16780-e569-11ea-8bde-91b6d28a63a2.png)

**Comment by ilyasst** [comment]
The code then plots the displacement or strain for a node this way:
![Figure](https://user-images.githubusercontent.com/443955/90989679-ab826e00-e569-11ea-9032-e8ac20419aea.png)

It would be nice to update the code so that it shows the graph this way:
![Capture_decran_2020-08-10_a_12 15 55](https://user-images.githubusercontent.com/443955/90989712-ff8d5280-e569-11ea-935f-5cd99360e24d.png)
I believe that @NicolasTHU you already have the code for the style of the plot.

**Comment by NicolasTHU** [comment]
Yes of course, I juste made an update of my previous PR and now there is a title on the plot and a legend on the curve #11

**Comment by NicolasTHU** [comment]
Concerning the node map, the modifications can be found in commit bde8daa
